plugins/src/plugins/gripper/gripper.cpp

```diff
@@ -76,14 +78,22 @@ void Gripper::Load(physics::ModelPtr _parent, sdf::ElementPtr /*_sdf*/)
   grabJoint->SetName("gripper_grab_puck");
   grabJoint->SetModel( model_);
   // grabJoint->SetPose(gazebo::math::Vector3(0.0,0.0,0.0));
+  //
+  
+  rnd_gen_=std::mt19937(time(0));
+  do_test_=std::uniform_real_distribution<double>(0,1);
+  oldTime_=model_->GetWorld()->GetSimTime();
 }
```

Comment:
Please add spaces around `=`: `rnd_gen_=std::mt19937(time(0));` should be `rnd_gen_ = std::mt19937(time(0));`